docs/meshstack.metadata-tags.md

```diff
@@ -183,7 +183,9 @@ This is useful if your organization needs additional metadata like e.g. contract
 
 ## Tags on meshUsers
 
-meshUsers are defined globally in meshStack and therefore their tags are identicial within all meshCustomers. As described in [meshPolicies](meshcloud.policies.md#meshpolicies-for-meshUsers/Groups) there are usecases where it makes sense to just apply a set of default tags to all meshUsers. I.e. you want to allow to assign all users to "dev" and "qa" projects, but not to "prod" projects.
+meshUsers are defined globally in meshStack and therefore their tags are identical within all meshCustomers. As described in [meshPolicies](meshcloud.policies.md#meshpolicies-for-meshUsers/Groups) there are use cases where it makes sense to just apply a set of default tags to all meshUsers. I.e. you want to allow to assign all users to "dev" and "qa" projects, but not to "prod" projects.
+
+The following configuration options are available at `mesh.web.user`:
```

Comment:
not sure why this change is in here but I'll allow it

Reply:
hm I think i just noticed some typos. `identicial` vs `identical`